### ISSUE TITLE
Doesn't work above 100 episodes

### ISSUE THREAD

**Issue by g0kuz** [software-2/MAL-Renamer#1]
Anything 100> will not appear. Hope you could fix it!

**Comment by software-2** [comment]
Yeah, I need to implement paging support. Unfortunately not a quick fix. It's a planned part of a larger overhaul.
